app/src/pages/inside/logsPage/historyLine/historyLineItem/historyLineItem.scss

```diff
@@ -17,6 +17,7 @@
 .history-line-item {
   position: relative;
   margin-right: 37px;
+  opacity: 0.3;
```

Comment:
Same. This opacity will only be used when the user opens the `Make Decision` modal with a new design. For now, you can skip it.